### DISCUSSION TITLE
Very nice OS, why not add min browser in the standard version

### DISCUSSION THREAD

**Discussion by cittadhammo** [minios-linux/minios-live#13] [Ideas]
Dear MiniOS,

I have been reviving a lenovo T61 with your OS and it is beautiful, even just on a permanent USB live stick. 

I have tried many different Browsers as this was the main limitation on the windows 10 version I had. 

The native firefox struggle a lot on the MiniOS because of the 2GB only of ram of the computer I believe. So I have tried :

- min browser (the best)
- netsurf-gtk (text + css + some js)
- dillo (text + css)
- lynx (text)

And https://minbrowser.org/ is simply the best and very modern all JS works flawlessly. I would encourage to have it build in, in addition (or in place) of firefox. This 2006 computer now can browse and run so nicely! 

Thank you so much !
Best wishes

**Comment by crim50n** [comment]
We will consider your suggestion and, if it does not result in a loss of functionality or increase the size of the distribution, we will replace the default browser. You can create a module with the browser you need by yourself using the instructions below.
https://github.com/minios-linux/minios-live/wiki/Creating-modules

**Reply by cittadhammo** [reply]
Ok good, thank you,

Also, it seems that there are still difficulties with integrated tools like google gmail and workspace. It is a security thing and non-mainstream browser seems to be black listed by the google service. https://github.com/minbrowser/min/issues/868

So needs testing before changing the default browser I would say ;-)

**Reply by crim50n** [reply]
I'll check it out
